resources/jmqttd/jmqttd.py

```diff
@@ -65,16 +68,83 @@ def __init__(self, queue, message):
 		if 'statustopic' in message:
 			self.mqttstatustopic = message['statustopic']
 
+		self.mqtttls = False
+		if 'tls' in message:
+			self.mqtttls = message['tls'] == '1'
+
+		self.mqtttlscafile = None
+		if 'tlscafile' in message and message['tlscafile'] != '':
+			if os.access(message['tlscafile'], os.R_OK):
```

Comment:
Would be better to make this check in cmd_handler - newMqttClient section (make controls before starts creation of jMQTTClient object like in subscribeTopic)